edx/edx-enterprise
# ENT-678 Fix enterprise logo stretching in enterprise sidebar

@asadiqbal08 @saleem-latif 

**Description:** Fix enterprise logo stretching issue in enterprise sidebar on course/program enrollment pages.

**JIRA:** [ENT-678](https://openedx.atlassian.net/browse/ENT-678)

**Dependencies:** N/A

**Merge deadline:** 15-1-2018

**Installation instructions:** Install edx-enterprise from this branch `zub/ENT-678-fix-enterprise-logo-streching` in `edx-platform`.

**Testing instructions:**

1. Create an enterprise with a logo.
2. As an enterprise learner login (from enterprise related SSO) and go to course enrollment page.
3. Verify that the enterprise logo is not stretched vertically. Max height and max width of enterprise logo will be `150px`.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.
